src/commands/projects/create/nosql.ts

```diff
@@ -28,6 +28,13 @@ export default class NosqlCommand extends AbstractProjectCreateCommand {
       exclusive: ['databaseConnectionURL'],
       required: false,
     }),
+    language: flags.string({
+      char: 'l',
+      description: 'In which language would you like to generate your sources?',
+      options: Object.values(languages).map(language => language.name),
+      default: 'required',
+      required: false,
```

Comment:
[Question] What are these parameters used for ? I though this flag would have been required, accordingly to the prompts